go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/kaspanet/faucet
 go 1.13
 
 require (
+	github.com/go-pg/pg/v9 v9.1.3
```

Comment:
Please run go mod tidy to make sure gorm is truly gone.

Reply:
Done already